code/modules/cargo/orderconsole.dm

```diff
@@ -302,16 +302,29 @@
 				say(blockade_warning)
 				return
 
-			//make an copy of the cart before its cleared by the shuttle
-			var/list/cart_list = list()
-			for(var/datum/supply_order/order in SSshuttle.shopping_list)
-				if(cart_list[order.pack.name])
-					cart_list[order.pack.name]["amount"]++
-					continue
-				cart_list[order.pack.name] = list(
-					"order" = order,
-					"amount" = 1
-				)
+			//create the paper from the SSshuttle.shopping_list
+			if(length(SSshuttle.shopping_list))
+				var/obj/item/paper/requisition_paper = new(get_turf(src))
+				requisition_paper.name = "requisition form"
+				var/requisition_text = "<h2>[station_name()] Supply Requisition</h2>"
+				requisition_text += "<hr/>"
+				requisition_text += "Time of Order: [station_time_timestamp()]<br/><br/>"
+				for(var/datum/supply_order/order in SSshuttle.shopping_list)
+					requisition_text += "<b>[order.pack.name]</b></br>"
+					requisition_text += "- Order ID: [order.id]</br>"
+					var/restrictions = "[SSid_access.get_access_desc(order.pack.access)]"
+					if(restrictions != "")
+						requisition_text += "- Access Restrictions: [restrictions]</br>"
+					requisition_text += "- Ordered by: [order.orderer] ([order.orderer_rank])</br>"
+					var/paying_account = "[order.paying_account]"
+					if(paying_account)
+						requisition_text += "- Paid Privatly by: [order.paying_account.account_holder]<br/>"
+					var/reason = "[order.reason]"
```

Comment:
can't you just do reason = order.reason? same with restrictions

Reply:
I can try it. Since I have no idea how the compiler handles String conversions nor how it handles String If-Statments. It just was the laziest and easiest way to handle the If-Statement, without the knowledge of how exactly Datums and Variables are handled by the compiler and engine.

Thanks for the valuable pointers ;)

Reply:
I think it's fine this way